Makefile

```diff
@@ -26,6 +26,9 @@ clean:
 	rm -rf viam-orb-slam3/ORB_SLAM3/Thirdparty/Sophus/build
 	rm -rf viam-orb-slam3/bin
 
+clean-all:
+	git clean -fxd
```

Comment:
[note to self] this is also in viam-cartographer https://github.com/viamrobotics/viam-cartographer/blob/b978910877dca17541f482ef3d0fe544ce3bd889/Makefile#L27

documentation for the flags: https://git-scm.com/docs/git-clean